project/plugins.sbt

```diff
@@ -4,6 +4,6 @@ resolvers ++= Seq(
     "OSS Sonatype Releases" at "https://oss.sonatype.org/content/repositories/releases/",
     "OSS Sonatype Snaps" at "https://oss.sonatype.org/content/repositories/snapshots/")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.0.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")
 
-addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.4.0")
\ No newline at end of file
+addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.7.0-SNAPSHOT")
```

Comment:
I just updated the version but actually I would recomment to remove it cause since IntelliJ Idea 13 sbt plugin sbt-idea is not widely used anymore (take a look at: https://github.com/paypal/cascade/pull/142).

So waiting for your yes and will remove it.